.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,104 @@
+name: Mac and Windows Builds
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+        python-version: ['3.5', '3.6', '3.7']  # We build 3.5 on Azure Pipelines
+        architecture: ['x86', 'x64']
+        exclude:
+          # Don't build 32-bit on Mac
+          - os: macos-latest
+            architecture: x86
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # This LOOKS like it is checking out 'master', but it is using the 'master' version of the checkout action
+      # It is actually checking out the most recent version on this branch
+      - name: Checkout
+        uses: actions/checkout@master
+
+      # Visual Studio 2019 (windows-latest) doesn't include these files for some reason??
+      # Even stranger, it really only affects Python 3.5, but we run it across all versions for consistency
+      # https://social.msdn.microsoft.com/Forums/vstudio/en-US/06675e6d-9a0b-46f5-8de0-10237fab1ece/link-error-lnk1158?forum=vcgeneral
+      - name: Fix Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          Copy-Item "C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\rc.exe" -Destination "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN"
+          Copy-Item "C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\rcdll.dll" -Destination "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN"
+        shell: powershell
+
+      - name: Setting up Python
+        uses: actions/setup-python@v1.1.1
```

Comment:
What is this doing?

Reply:
This is a quirk of GitHub Actions. They are essentially trying to make it so you can use other people's action in your workflow. In this case, to set up the correct Python (Version and Architecture), we have to use the [setup-python action](https://github.com/actions/setup-python). This is a first-party action from GitHub, so it should be safe